trans/TeXworks_de.ts

```diff
@@ -362,8 +362,7 @@ Wollen Sie fortfahren?</translation>
     </message>
     <message>
         <location filename="../src/PDFDocumentWindow.cpp" line="1026"/>
-        <source>Unfortunately, this version of %1 is unable to print PDF documents due to various technical reasons.
-</source>
+        <source>Unfortunately, this version of %1 is unable to print PDF documents due to various technical reasons.</source>
         <translation>Diese Version von %1 kann aufgrund verschiedener technischer Gründe leider keine PDF Dokumente drucken.</translation>
```

Comment:
I think the newline at the end of the source text should persist (it is in the source code after all!). It should, however, also be included in the translation (to ensure the correct white spacing in the dialog).